ckanext/openafrica/lib/helpers.py

```diff
@@ -1,5 +1,14 @@
 import datetime
+from ckan.common import config
 
 def current_year():
     year = datetime.datetime.now().year
-    return year    
+    return year
+
+def enable_datarequest():
```

Comment:
I believe this can be better named and easily reusable by having it as `def is_plugin_enabled(plugin_name)`.

Reply:
Also, let's add docstrings about this.

Reply:
Thank you for that. I have fixed it.